pom.xml

```diff
@@ -955,6 +950,97 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
@prondzyn I think we should put this as a plugin in `site` profile, same as Cobertura above, instead of having a separate profile.